BlueprintUI/Sources/Element/ElementContent.swift

```diff
@@ -98,27 +89,41 @@ extension ElementContent {
         init(layout: LayoutType) {
             self.layout = layout
         }
+        
+        /// Adds the given child element.
+        public mutating func add(element: Element, traits: LayoutType.Traits = LayoutType.defaultTraits, key: AnyHashable? = nil) {
```

Comment:
While I’m thinking about it: would be maybe nice to add a convenience `func add(elements: [Element])` that uses default traits and no keys, since that’s a common use-case for these from my quick code scan.